packages/lumx-react/src/components/flag/Flag.tsx

```diff
@@ -28,17 +30,19 @@ const DEFAULT_PROPS: Partial<FlagProps> = {
  * @return React element.
  */
 export const Flag: Comp<FlagProps, HTMLDivElement> = forwardRef((props, ref) => {
-    const { label, icon, color, className, theme, ...forwardedProps } = props;
+    const { label, icon, color, className, theme, truncate, ...forwardedProps } = props;
```

Comment:
Naive question, why don't you use a default value `false` here to avoid using `!!truncate` then?

Reply:
But I should have done that in the classes too 🤔

Reply:
But it's typed as boolean, they can't unless they ignore the typing issues